### PR TITLE
Add a command to list processes

### PR DESCRIPTION
**Pull request by gssbzn**
## Proposed changes

Looks like atlas changed how we generate the id for the atlas host so I needed this command to get the new id format

## Checklist

- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added any necessary documentation (if appropriate)
- [x] I have run `make fmt` and formatted my code

## Further comments

```bash
mongocli atlas processes ls
[
	{
		"created": "2020-04-15T15:44:15Z",
		"groupId": "5e429f2e06822c6eac4d59c9",
		"hostname": "atlas-k11skw-shard-00-00.vnp8x.azure.mongodb-dev.net",
		"id": "atlas-k11skw-shard-00-00.vnp8x.azure.mongodb-dev.net:27017",
		"lastPing": "2020-04-15T17:12:55Z",
		"links": [
			{
				"rel": "self",
				"href": "https://cloud-dev.mongodb.com/api/atlas/v1.0/groups/5e429f2e06822c6eac4d59c9/processes/atlas-k11skw-shard-00-00.vnp8x.azure.mongodb-dev.net:27017"
			}
		],
		"port": 27017,
		"shardName": "",
		"replicaSetName": "atlas-k11skw-shard-0",
		"typeName": "REPLICA_PRIMARY",
		"version": "4.0.17"
	},
	{
		"created": "2020-04-15T15:44:15Z",
		"groupId": "5e429f2e06822c6eac4d59c9",
		"hostname": "atlas-k11skw-shard-00-01.vnp8x.azure.mongodb-dev.net",
		"id": "atlas-k11skw-shard-00-01.vnp8x.azure.mongodb-dev.net:27017",
		"lastPing": "2020-04-15T17:12:55Z",
		"links": [
			{
				"rel": "self",
				"href": "https://cloud-dev.mongodb.com/api/atlas/v1.0/groups/5e429f2e06822c6eac4d59c9/processes/atlas-k11skw-shard-00-01.vnp8x.azure.mongodb-dev.net:27017"
			}
		],
		"port": 27017,
		"shardName": "",
		"replicaSetName": "atlas-k11skw-shard-0",
		"typeName": "REPLICA_SECONDARY",
		"version": "4.0.17"
	},
	{
		"created": "2020-04-15T15:44:15Z",
		"groupId": "5e429f2e06822c6eac4d59c9",
		"hostname": "atlas-k11skw-shard-00-02.vnp8x.azure.mongodb-dev.net",
		"id": "atlas-k11skw-shard-00-02.vnp8x.azure.mongodb-dev.net:27017",
		"lastPing": "2020-04-15T17:12:55Z",
		"links": [
			{
				"rel": "self",
				"href": "https://cloud-dev.mongodb.com/api/atlas/v1.0/groups/5e429f2e06822c6eac4d59c9/processes/atlas-k11skw-shard-00-02.vnp8x.azure.mongodb-dev.net:27017"
			}
		],
		"port": 27017,
		"shardName": "",
		"replicaSetName": "atlas-k11skw-shard-0",
		"typeName": "REPLICA_SECONDARY",
		"version": "4.0.17"
	}
]
```

### CODE REVIEW

File: internal/cli/atlas.go [hunk 1]
```diff
@@ -31,6 +31,8 @@ func AtlasBuilder() *cobra.Command {
 	cmd.AddCommand(AtlasBackupsBuilder())
 	cmd.AddCommand(AtlasEventsBuilder())
 	cmd.AddCommand(AtlasMeasurementsBuilder())
+	// cmd.AddCommand(AtlasLogsBuilder())
```
Comment (colm-quinn) [inline internal/cli/atlas.go:34]: [np] Can be removed now? Not blocking

File: internal/description/description.go [hunk 2]
```diff
@@ -43,6 +44,8 @@ const (
 	DescribeCluster        = "Describe a cluster."
 	ListClusters           = "List clusters for a project."
 	UpdateCluster          = "Update a MongoDB cluster in Atlas."
+	Processes              = "Manage processes for your project."
```
Comment (robcarlan-mlab) [inline internal/description/description.go:47]: Should the language here always refer to `a project`?

File: internal/cli/atlas.go (hunk 1 above)
Comment (gssbzn) [reply internal/cli/atlas.go:34]: That's what I was working on and could not use as I needed the new format, I'll merge away as the next PR will remove the comments

File: internal/description/description.go (hunk 2 above)
Comment (gssbzn) [reply internal/description/description.go:47]: This format was done as a favour from the docs team, I'll ask them to give this list of descriptions another go

**Pull request merged**